### PR TITLE
Bug 1869596: fix pipeline task sidebar form error

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://issues.redhat.com/browse/ODC-4511

**Root Analysis:**
The pipeline task sidebar shows `Required` field error for non-empty field

**Solution Description:**
Fixed the field level validation

**GIF:**
tasksidebarerror

### CODE REVIEW

File: frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarParam.tsx [hunk 1]
```diff
@@ -5,25 +5,29 @@ import { taskParamIsRequired } from '../utils';
...
 const TaskSidebarParam: React.FC<TaskSidebarParamProps> = (props) => {
-  const { hasParamError, onChange, resourceParam, taskParam } = props;
+  const { onChange, resourceParam, taskParam } = props;
   const [dirty, setDirty] = React.useState(false);
 
   const currentValue = taskParam?.value;
   const emptyIsInvalid = taskParamIsRequired(resourceParam);
 
-  const isValid = !(dirty && hasParamError && emptyIsInvalid && currentValue != null);
+  const isValid = () => {
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarParam.tsx:20]: Why a function and not just a straight boolean?

**Comment by abhinandan13jan** [comment]
ran fine locally and error is fixed
/lgtm

**Comment by abhinandan13jan** [comment]
is this expected?
Screencast from 08-20-2020 06_36_15 PM

/lgtm cancel

**Pull request merged**